GOVERNANCE.md

```diff
@@ -0,0 +1,81 @@
+# Abstract
+
+The purpose of this document is to formalize the governance process used by the
+tskit-dev community for core software projects, to clarify who has what responsibilities,
+and how decisions are made on changes to those responsibilities.
+
+tskit-dev is an open and inclusive community. Anyone with an interest in the
+software and science can contribute discussion, and propose changes to code, documentation
+or website content.
+
+# Scope
+
+The governance model described in this document applies to the core
+repositories of the tskit software ecosystem. These are:
+
+- [tskit - C and Python API](http://github.com/tskit-dev/tskit)
+- [msprime - coalescent simulator](http://github.com/tskit-dev/msprime)
+- [tszip - compression library for tskit](http://github.com/tskit-dev/tszip)
+- [tskit-site - website for the ecosystem](http://github.com/tskit-dev/tskit-site)
+- [kastore - key-value store](http://github.com/tskit-dev/kastore)
+- [administrative](http://github.com/tskit-dev/administrative)
+- [.github - common config](http://github.com/tskit-dev/.github)
+
+
+# Roles
+
+## Contributors
+
+Anyone can become a contributor to the tskit-dev project by, for example,
+
+- proposing, discussing, or reviewing a change to the code, documentation, or specification
+  via a GitHub pull request to the above repositories;
+- reporting a GitHub issue or starting a discussion on the above repositories;
```

Comment:
What's the process for someone getting added to the github org? (I don't know, just asking the question for visibility)

Reply:
We need to set this up and have a clean out of permissions across the repos. It's under settings on the org page.

Reply:
I meant what's the process for deciding whether someone should be in the Org or not. That's something we should document

Reply:
sounds good but in vague terms; too prescriptive might go out of date (as for instance perhaps possible github roles change)

perhaps some general principles - like "the steering committee will choose roles in the github org so that X Y and Z"

Reply:
ah I see this has been added below